plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -58,6 +58,10 @@ import javax.annotation.Nullable;
 	<#assign extendsClass = "TamableAnimal">
 </#if>
 
+<#if data.spawnInRaids>
+	<#assign extendsClass = "Raider">

```

Comment:
What happens if I make the mob also `tameable` and `breakable`? I won't have the class extending `TamableAnimal` and I'll loose methods inherited by it. This feels wrong... Did you test making a mob tameable&breedable that can be in raids?

Reply:
i.e. I guess this is why it doesn't build?

Reply:
No, is there a way to extend the class to both Raider and TamableAnimal/Animal?

Reply:
No you can't! In Java you can't extend two classes...

Reply:
Fixed